javascript/javascript1/week4/hw4.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="hw4.js"></script>
+    <title>Document</title>
+</head>
+
+<body>
+
```

Comment:
Did you forget to add a JS script here? 🤔

Reply:
I just used it in the head instead of body.